evm/utils.js

```diff
@@ -484,18 +484,18 @@ const getDeployedAddress = async (deployer, deployMethod, options = {}) => {
     }
 };
 
-const getProxy = async (config, chain) => {
-    const address = (await httpGet(`${config.axelar.lcd}/axelar/evm/v1beta1/gateway_address/${chain}`)).address;
+const getProxy = async (axelarConfig, chain) => {
```

Comment:
```suggestion
const getProxy = async (axelar, chain) => {
```
could just name it `axelar` for brevity similar to how we name `chain` instead of `chainConfig`